python/ray/data/_internal/execution/resource_manager.py

```diff
@@ -28,16 +28,6 @@ class ResourceManager:
     # The interval in seconds at which the global resource limits are refreshed.
     GLOBAL_LIMITS_UPDATE_INTERVAL_S = 10
 
-    # The fraction of the object store capacity that will be used as the default object
-    # store memory limit for the streaming executor,
-    # when `ReservationOpResourceAllocator` is enabled.
-    DEFAULT_OBJECT_STORE_MEMORY_LIMIT_FRACTION = 0.5
-
-    # The fraction of the object store capacity that will be used as the default object
-    # store memory limit for the streaming executor,
-    # when `ReservationOpResourceAllocator` is not enabled.
-    DEFAULT_OBJECT_STORE_MEMORY_LIMIT_FRACTION_WO_RESOURCE_RESERVATION = 0.25
```

Comment:
keep them, if the override is set, use the override regardless of `self.op_resource_allocator_enabled`, otherwise, use them

Reply:
done